site/src/pages/docs/toc.json

```
@@ -24,7 +24,8 @@
     "server-sse",
     "server-service-registration",
     "server-multipart",
-    "server-timeouts"
+    "server-timeouts",
+    "server-flags-provider"
```

What do you think of moving this to the advanced section this this isn't really specific to server-side?

```suggestion
    "advanced-flags-provider"
```

---

Thanks, I agree with it.